### ISSUE TITLE
Question about quicksearch

### ISSUE THREAD

**Issue by cervedgroup** [osTicket/osTicket#5648]
For a couple or reasons, we are on v1.10.7 version , and we can't upgrade ...

I need to modify the search that start when an operator type something in the quick search box 
![image](https://user-images.githubusercontent.com/6535381/95676209-c3b35880-0bbc-11eb-9494-f4ca3b49a124.png)

because is very cpu consuming (i need to modify in order to avoid a full text search in all tickets) , can you please tell me where is the query made for that kind of search ?

Thank you very much

**Comment by JediKev** [comment]
@cervedgroup start with `include/ajax.tickets.php`

**Comment by cervedgroup** [comment]
@JediKev thank you, i've commented out that two lines

elseif (!count($hits) && preg_match('`\w$`u', $q)) {
            // Do wild-card fulltext search
            **//$_REQUEST['q'] = $q.'*';
            //return $this->lookup();**
        }
let see if goes better

**Comment by cervedgroup** [comment]
no way ... seems that the typehead search locks some table and that cause a serious delay editing tickets / replaying etc..
any suggestion to disable the typehead / or to remove the box and leave only the advanced search ?

**Comment by cervedgroup** [comment]
that kind of query is causing the issue

`SELECT distinct f.id, COALESCE(a.type, f.ft), a.`name` FROM ost_file f LEFT JOIN ost_attachment a ON (a.file_id = f.id)
            WHERE ((a.`type` IN ('C', 'F', 'T', 'P') AND a.`inline` = 1) OR f.ft = 'L') AND f.`type` LIKE 'image/%' UNION SELECT f.id, a.`type`, a.`name` FROM ost_thread t
                JOIN ost_thread_entry th ON (th.thread_id = t.id)
                JOIN ost_attachment a ON (a.object_id = th.id AND a.`type` = 'H')
                JOIN ost_file f ON (a.file_id = f.id)
                WHERE a.`inline` = 1 AND t.id=420895`

this query is running since 20 minutes

```
SELECT distinct f.id, COALESCE(a.type, f.ft), a.`name` FROM ost_file f LEFT JOIN ost_attachment a ON (a.file_id = f.id)
            WHERE ((a.`type` IN ('C', 'F', 'T', 'P') AND a.`inline` = 1) OR f.ft = 'L') AND f.`type` LIKE 'image/%' UNION SELECT f.id, a.`type`, a.`name` FROM ost_thread t
                JOIN ost_thread_entry th ON (th.thread_id = t.id)
                JOIN ost_attachment a ON (a.object_id = th.id AND a.`type` = 'H')
                JOIN ost_file f ON (a.file_id = f.id)
                WHERE a.`inline` = 1 AND t.id=420895
```

**Comment by cervedgroup** [comment]
the funny thing is that ticket with ticket_id = 420895 don't exists in ost_ticket table ...

**Comment by nicoletta-maia** [comment]
Hi @cervedgroup, 420895 is the thread id (`ost_thread`.`id`)

**Comment by cervedgroup** [comment]
@nicoletta-maia 
yes i've found the file causing issue, but is a zip file of 10kb in size . Our ost_file_chunk is around 45 GB maybe that is the problem ?
